docs/validation-rules.rst

```diff
@@ -444,7 +444,8 @@ The assigned data can be of any type.
 min, max
 --------
 
-Minimum and maximum value allowed for any types that implement comparison operators.
+Minimum and maximum value allowed for any types that implement comparison operators. See
```

Comment:
The proposal seems fine, but not sure what are ` (__lt__ or __gt__)`, I can just add them if you want. I just felt that by comparison operators we are going for `\*of-rules`

Roger that, a simple example coming right up.

Reply:
these "dunder" (from double underscore) or also known as "magic" methods do implement these *operations* that the *operators* imply. so, maybe just "comparison operations" instead of "operator"?

Reply:
Thanks, @funkyfuture for the explanation.